samples/mpsl/timeslot/src/main.c

```diff
@@ -84,10 +106,16 @@ static mpsl_timeslot_signal_return_param_t *mpsl_timeslot_callback(
 			signal_callback_return_param.params.request.p_next =
 				NULL;
 			signal_callback_return_param.callback_action =
-				MPSL_TIMESLOT_SIGNAL_ACTION_NONE;
+				MPSL_TIMESLOT_SIGNAL_ACTION_END;
 		}
 
 		p_ret_val = &signal_callback_return_param;
+
+		/* Cleanup TIMER0 IRQ settings */
+		nrf_timer_int_disable(NRF_TIMER0, NRF_TIMER_INT_COMPARE0_MASK);
```

Comment:
Better to clean the event too.

Reply:
I'm not clear what you mean here.  Please elaborate.

Reply:
updated